napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -808,7 +813,9 @@ def _add_to_installed(distname, enabled, npe_version=1):
             if distname in self.already_installed or distname == 'napari':
                 continue
             enabled = not pm2.is_disabled(manifest.name)
-            _add_to_installed(distname, enabled, npe_version=2)
+            # if it's an Npe1 adaptor, call it v1
+            npev = 'shim' if 'npe1' in type(manifest).__name__.lower() else 2
```

Comment:
what is `type(manifest).__name__.lower()` actually checking sorry? Just curious how we're discerning between the different manifests, I have no issue with it.

Reply:
it's checking the name of the class, to make sure it's `NPE1Adapter` :joy: ... it could be an `isinstance`, but requires an import from private module.  I dunno 🤷